pkg/cli/extract/extract.go

```diff
@@ -23,30 +23,30 @@ import (
 
 var (
 	extractLong = templates.LongDesc(`
-		Extract files out of secrets and config maps
+		Extract files out of secrets and config maps.
 
 		The extract command makes it easy to download the contents of a config map or secret into a directory.
 		Each key in the config map or secret is created as a separate file with the name of the key, as it
 		is when you mount a secret or config map into a container.
 
-		You may extract the contents of a secret or config map to standard out by passing '-' to --to. The
-		names of each key will be written to stdandard error.
+		You may extract the contents of a secret or config map to standard out by passing '-' to --. The
```

Comment:
@soltysh Can you make sure that the removal of the second "to" is correct here?

Reply:
No, `--to` is a flag so it has to stay as is.